hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/ha/OMFailoverProxyProvider.java

```diff
@@ -84,16 +86,23 @@ public OMFailoverProxyProvider(OzoneConfiguration configuration,
   public final class OMProxyInfo
       extends FailoverProxyProvider.ProxyInfo<OzoneManagerProtocolPB> {
     private InetSocketAddress address;
+    private Text dtService;
```

Comment:
During failover Om proxy may change and point to another OM instance, in that case our previously set service field will become unusable.

Reply:
The dtService will be a combined URI format that contains both instances' ip:port. The token selector will be updated to accept.

Reply:
oh, you mean in follow up jira it will be uri. Make sense.